### PR TITLE
Release/snowplow web/0.6.0

### PR DESCRIPTION
**Pull request by emielver**
## Description & motivation
Release 0.6.0 (Waiting for integration tests to pass then dropping one commit) 
Integration tests passed (✅ )

## Checklist
- [x] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)

### CODE REVIEW

File: models/sessions/scratch/snowplow_web_sessions_this_run.sql [hunk 1]
```diff
@@ -30,7 +30,7 @@ select
 
   {% if var('snowplow__session_stitching') %}
     -- updated with mapping as part of post hook on derived sessions table
-    a.domain_userid as stitched_user_id, 
+    cast(a.domain_userid as {{dbt_utils.type_string() }}) as stitched_user_id,
```
Comment (bill-warner) [inline models/sessions/scratch/snowplow_web_sessions_this_run.sql:33]: I noticed during testing this doesn't actually increase the char limit for Redshift, at least when moving from the old version to the new (I haven't tested a full refresh). I am wondering is it worth writing our own version of this macro with the ability to specify the char limit

**Review by bill-warner — COMMENTED** [review]
Looks great. Spotted one issue around the char limit for the `stitched_user_id` for RS. Would be good to resolve this before the release. 

Just a reminder to update the CHANGELOG date and to call out the change in cluster macro names in the release notes

File: models/sessions/scratch/snowplow_web_sessions_this_run.sql (hunk 1 above)
Comment (bill-warner) [reply models/sessions/scratch/snowplow_web_sessions_this_run.sql:33]: Just tested a full refresh. Same result. Still set to 128

Comment (emielver) [reply models/sessions/scratch/snowplow_web_sessions_this_run.sql:33]: I'll have a look at implementing this today in snowplow utils:
https://github.com/snowplow/dbt-snowplow-utils/issues/65

**Review by bill-warner — APPROVED** [review]
LGTM aside from splitting up that latest commit. Great work on this. Once that is done I think we are good to release. I would suggest we release the utils package and mobile first then once they are in dbt's hub we can see the int tests pass on package, then release. WDYT?

**Pull request merged**